docs/content/getting-started/quickstart.mdx

```diff
@@ -0,0 +1,198 @@
+
+<Image
+alt="HackerNews assets in Dagster's Asset Graph, unmaterialized"
+src="/images/getting-started/quickstart/quickstart-unmaterialized.png"
+width={2000}
+height={816}
+/>
+
+Congratulations! You have successfully materialized two Dagster assets:
+
+<Image
+alt="HackerNews asset graph"
+src="/images/getting-started/quickstart/quickstart.png"
+width={2000}
+height={1956}
+/>
+
+But wait - there's more. Because the `hackernews_top_stories` asset returned some `metadata`, you can view the metadata right in the UI:
+
+1. Click the asset
+2. In the sidebar, click the **Show Markdown** link in the **Materialization in Last Run** section. This opens a preview of the pipeline result, allowing you to view the top 10 HackerNews stories:
+
+<Image
+alt="Markdown preview of HackerNews top 10 stories"
+src="/images/getting-started/quickstart/hn-preview.png"
+width={2000}
+height={1754}
+/>
+
+## Understanding the Code
+
+The quickstart project defines two **Assets** using the `@asset` decorator:
+
+- `hackernews_top_story_ids` retrieves the top stories from the Hacker News API and saves them as a JSON file.
+- `hackernews_top_stories` builds upon the first asset, retrieving data for each story and saving it as a CSV file.
+
+```python file=/getting-started/quickstart/assets.py
+import json
+
+import pandas as pd
+import requests
+
+from dagster import (
+    MaterializeResult,
+    MetadataValue,
+    asset,
+)
+
+from .configurations import HNStoriesConfig
+
+
+@asset
+def hackernews_top_story_ids(config: HNStoriesConfig):
+    """Get top stories from the HackerNews top stories endpoint."""
+    top_story_ids = requests.get(
+        "https://hacker-news.firebaseio.com/v0/topstories.json"
+    ).json()
+
+    with open(config.hn_top_story_ids_path, "w") as f:
+        json.dump(top_story_ids[: config.top_stories_limit], f)
+
+
+@asset(deps=[hackernews_top_story_ids])
+def hackernews_top_stories(config: HNStoriesConfig) -> MaterializeResult:
+    """Get items based on story ids from the HackerNews items endpoint."""
+    with open(config.hn_top_story_ids_path, "r") as f:
+        hackernews_top_story_ids = json.load(f)
+
+    results = []
+    for item_id in hackernews_top_story_ids:
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
+        results.append(item)
+
+    df = pd.DataFrame(results)
+    df.to_csv(config.hn_top_stories_path)
+
+    return MaterializeResult(
+        metadata={
+            "num_records": len(df),
+            "preview": MetadataValue.md(str(df[["title", "by", "url"]].to_markdown())),
+        }
+    )
+```
+
+The `hackernews_top_stories` asset returns a `MaterializeResult` with metadata, which is displayed in the Dagster UI, providing a markdown preview of the top stories.
```

Comment:
This feels like it's in a weird place - thoughts on combining it with the other sentence about this asset that comes before the code?

Reply:
Good suggestion, thanks.